dist/docker/Dockerfile.alpine

```diff
@@ -45,11 +45,14 @@ RUN pip3 install --upgrade pip && \
 # Build and Install Cloe
 WORKDIR /cloe
 COPY . /cloe
+ARG WITH_VTD=0
+ARG PACKAGE_TARGET=package-select
```

Comment:
I think this is not going to work, s. our previous discussion [here](https://github.com/eclipse/cloe/pull/24#discussion_r582051715). There will be conflicting boost versions used for oak and cloe-runtime.

Reply:
Right you are. I created a new target, `package-auto`, that builds all selected packages together.